cost-analyzer/values-agent.yaml

```diff
@@ -0,0 +1,72 @@
+# Kubecost running as an Agent is designed for external hosting. The current setup deploys a
+# kubecost-agent pod, low data retention prometheus server + thanos sidecar, and node-exporter. 
+global:
+  thanos:
+    enabled: false 
+  grafana:
+    enabled: false 
+    proxy: false 
+
+# Agent enables specific features designed to enhance the metrics exporter deployment
+# with enhancements designed for external hosting. 
+agent: true 
+
+# Exporter Pod
+kubecostMetrics: 
+  exporter:
+    enabled: true 
+
+# Prometheus defaults to low rentention (10h), disables KSM, and attaches a thanos-sidecar
+# for exporting metrics. 
+prometheus:
+  kube-state-metrics:
+    enabled: false
+    disabled: true
```

Comment:
This is how you disable kube-state-metrics. 

![bitmoji](https://sdk.bitmoji.com/render/panel/5087538c-5446-4385-b4d5-d57621aa47f2-3ab2de09-1581-4385-91ea-74299b921b9a-v1.png?transparent=1&palette=1&width=246)

Reply:
:man_facepalming: